project/AppDependencies.scala

```diff
@@ -19,29 +19,25 @@ import play.sbt.PlayImport._
 import sbt._
 
 object AppDependencies {
-  val bootstrapVersion = "7.21.0"
-  val mongoVersion = "1.3.0"
-  val playVersion = "play-28"
+  val bootstrapVersion = "8.4.0"
+  val mongoVersion = "1.7.0"
+  val playVersion = "play-30"
 
   val compile: Seq[ModuleID] = Seq(
     ws,
-    "uk.gov.hmrc"       %%  "bootstrap-backend-play-28"     % bootstrapVersion,
-    "uk.gov.hmrc"       %%  "domain"                        % s"8.3.0-$playVersion",
-    "uk.gov.hmrc"       %%  "play-hmrc-api"                 % s"7.2.0-$playVersion",
-    "uk.gov.hmrc"       %%  "play-hal"                      % s"3.4.0-$playVersion",
-    "uk.gov.hmrc"       %%  "mongo-feature-toggles-client"  % "0.4.0",
-    ehcache
+    "uk.gov.hmrc"       %%  s"bootstrap-backend-$playVersion"                   % bootstrapVersion,
+    "uk.gov.hmrc"       %%  s"domain-$playVersion"                              % "9.0.0",
+    "uk.gov.hmrc"       %%  s"play-hal-$playVersion"                            % "4.0.0",
+    "uk.gov.hmrc"       %%  s"mongo-feature-toggles-client-$playVersion"        % "1.1.0",
```

Comment:
Just double check that mongo-feature-toggles-client doesn't use a different version of bootstrap or other libs as this can cause breaks as it doesn't know which version to use.